ts/Core/Axis/LogarithmicAxis.ts

```diff
@@ -204,10 +209,26 @@ class LogarithmicAxisAdditions {
     }
 
     public lin2log(num: number): number {
+        if (this.axis.options.allowNegativeLog) {
```

Comment:
I think this should be reverted, see https://github.com/highcharts/highcharts/issues/13914#issuecomment-671298850 . @bre1470 .